src/modules/vast.js

```diff
@@ -1,6 +1,31 @@
 // VAST support module
 'use strict';
 export default function (playerInstance, options) {
+    /**
+     * Gets CTA parameters from VAST and sets them on tempOptions
+     *
+     * @param {HTMLElement} titleCtaElement
+     *
+     * @param {any} tmpOptions
+     */
+    playerInstance.setCTAFromVast = (titleCtaElement, tmpOptions) => {
+        if (playerInstance.displayOptions.vastOptions.adCTATextVast && titleCtaElement) {
+            const mobileText = playerInstance.extractNodeDataByTagName(titleCtaElement, 'MobileText');
+            const desktopText = playerInstance.extractNodeDataByTagName(titleCtaElement, 'PCText');
+            const link = playerInstance.extractNodeDataByTagName(titleCtaElement, 'Link');
+            const tracking = playerInstance.extractNodeDataByTagName(titleCtaElement, 'Tracking');
+            const isMobile = window.matchMedia('(max-width: 768px)').matches;
+
+            if (mobileText && desktopText && link && tracking) {
```

Comment:
what if the mobileText is empty? 
```suggestion
            if (desktopText && link && tracking) {
```